test/framework/src/main/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -1809,10 +1811,6 @@ public String toString() {
         }
     }
 
-    protected static String formatted(String string, Object... args) {
-        return String.format(Locale.ROOT, string, args);
-    }
-
```

Comment:
You need retain this method (it is causing CI failures). Otherwise the changes seem to be good.

Reply:
Fixed.